sdks/java/io/hbase/src/main/java/org/apache/beam/sdk/io/hbase/HBaseIO.java

```diff
@@ -173,33 +182,33 @@ public static Read read() {
     /** Reads from the HBase instance indicated by the* given configuration. */
     public Read withConfiguration(Configuration configuration) {
       checkArgument(configuration != null, "configuration can not be null");
-      return new Read(new SerializableConfiguration(configuration), tableId, serializableScan);
+      return new Read(new Configuration(configuration), tableId, scan);
     }
 
     /** Reads from the specified table. */
     public Read withTableId(String tableId) {
       checkArgument(tableId != null, "tableIdcan not be null");
```

Comment:
typo "tableIdcan"

Reply:
fixing it, actually they were more ocurrencdes of this 'can' mistake I fixed them all now.

Reply:
thanks